application/frontend/tests/auth.setup.ts

```diff
@@ -1,13 +1,43 @@
-import { test as setup } from "@playwright/test"
+import { test as setup, expect } from "@playwright/test"
 import { firstSuperuser, firstSuperuserPassword } from "./config.ts"
 
 const authFile = "playwright/.auth/user.json"
 
 setup("authenticate", async ({ page }) => {
   await page.goto("/login")
+  await page.waitForLoadState("networkidle")
+
+  // Disable animations
+  await page.addStyleTag({
+    content: `
+      *,
+      *::before,
+      *::after {
+        transition: none !important;
+        animation: none !important;
+      }
+    `,
+  })
+
+  // Ensure the login form is present
+  await expect(page.getByPlaceholder("Email")).toBeVisible()
+  await expect(page.getByPlaceholder("Password")).toBeVisible()
+
+  // Fill in the form
   await page.getByPlaceholder("Email").fill(firstSuperuser)
   await page.getByPlaceholder("Password").fill(firstSuperuserPassword)
-  await page.getByRole("button", { name: "Log In" }).click()
+
+  // Ensure the "Log In" button is visible and enabled
+  const loginButton = page.getByRole("button", { name: "Log In" })
+  await expect(loginButton).toBeVisible()
+  await expect(loginButton).toBeEnabled()
+
+  // Click the "Log In" button
+  await loginButton.click()
```

Comment:
The test fails precisely because the elements (in this case, the login button) are not [Stable](https://playwright.dev/docs/actionability#stable).

Simply force the click by passing `{force: true}`. See: https://playwright.dev/docs/input#forcing-the-click. 
```suggestion
  await loginButton.click({force: true})
```

Reply:
currently on each navigation of the webpage I have the animations stopped before the click action. on the next sprint I can go and alter all of these click function lines to do what you have recommended.